cmd/broker/filter/main.go

```diff
@@ -56,8 +57,10 @@ const (
 type envConfig struct {
 	Broker        string `envconfig:"BROKER" required:"true"`
 	Namespace     string `envconfig:"NAMESPACE" required:"true"`
-	PodName       string `split_words:"true" required:"true"`
 	ContainerName string `split_words:"true" required:"true"`
+
+	// TODO: change this environment variable to something like "PodGroupName".
+	PodName string `split_words:"true" required:"true"`
```

Comment:
Can we keep the struct fields in their original order? see no reason to change, just add the comment.